openapi.yaml

```diff
@@ -37,7 +35,7 @@ paths:
                 items:
                   type: string
         '500':
-          description: Error
+          description: Internal Server Error
```

Comment:
As discussed on the review call - can you document the input to the API calls more detail.  Highlight what the infixes and such are and how they will be used

Reply:
Added comments and better descriptions for the CPS calls.